src/main/java/delete_post/interface_adapters/DeletePostViewModel.java

```diff
@@ -0,0 +1,4 @@
+package delete_post.interface_adapters;
+
```

Comment:
I don't think a viewmodel is necessary for this use case, it seems like it will mostly be interacting w the view post viewmodel